src/VirtoCommerce.NotificationsModule.Core/Model/Result/OperationResult.cs

```diff
@@ -0,0 +1,17 @@
+namespace VirtoCommerce.NotificationsModule.Core.Model.Result
+{
+    public abstract class OperationResult
+    {
+    }
+
+    public class ErrorResult : OperationResult
```

Comment:
Could you use one class for returning the result with IsSuccess and Errors collection, like e.g.
https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authorization.authorizationresult?view=aspnetcore-3.1.
I do not see the real need to have 3 classes here instead of one.

Reply:
Good point, but this code is reverted.